admin/tests/unit/routes/authenticated/users/list_test.js

```diff
@@ -45,11 +46,13 @@ module('Unit | Route | authenticated/users/list', function (hooks) {
     module('when queryParams filters are truthy', function () {
       test('it should call store.query with filters containing trimmed values', async function (assert) {
         // given
+        params.id = '123456';
```

Comment:
Pour avoir un test qui couvre les règles métier, je pense que tu devrais faire ceci

```suggestion
        params.id = ' 123  456     ';
```

Reply:
j'avais complètement oublié de prendre en compte l'espace entre chaque chiffre merci!